lib/adapters/translate_encoder.js

```diff
@@ -30,6 +30,8 @@ var encoder = processor({
         return null;
       case 'boolean':
         return val;
+      case 'uuid':
```

Comment:
I think you can get away with just adding this to the first case above, the big one. Maybe we should just move that to the default case? it would have allowed you to do this without any changes if it was already that way